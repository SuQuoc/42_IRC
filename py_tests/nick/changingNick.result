:AfterLife 001 client0 :Welcome to the Internet Relay Network, user0
:AfterLife 005 client0 NICKLEN=9 CHANTYPES=#& CHANMODES=oiklt :are supported by this server
:AfterLife 001 client1 :Welcome to the Internet Relay Network, user1
:AfterLife 005 client1 NICKLEN=9 CHANTYPES=#& CHANMODES=oiklt :are supported by this server
:AfterLife 001 client2 :Welcome to the Internet Relay Network, user2
:AfterLife 005 client2 NICKLEN=9 CHANTYPES=#& CHANMODES=oiklt :are supported by this server
:client1!user1@10.11.2.8 PRIVMSG client0 :You should see this message
:client0!user0@10.11.2.8 NICK :newNick
:client1!user1@10.11.2.8 PRIVMSG newNick :if you see this, it means that the nick was changed successfully
:AfterLife 401 client1 client0 :No such nick/channel
:client2!user2@10.11.2.8 NICK :client0
:AfterLife 433 client0 newNick :Nickname is already in use
:client1!user1@10.11.2.8 PRIVMSG client0 :if you see this, it means that the nick was changed successfully
