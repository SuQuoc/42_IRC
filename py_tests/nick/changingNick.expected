:AfterLife 001 client0 :Welcome to the Internet Relay Network, user0
:AfterLife 001 client1 :Welcome to the Internet Relay Network, user1
:AfterLife 001 client2 :Welcome to the Internet Relay Network, user2
:client1!user1@10.14.3.10 PRIVMSG client0 :You should see this message
:client0 NICK newNick
:client1!user1@10.14.3.10 PRIVMSG newNick :if you see this, it means that the nick was changed successfully
:AfterLife 401 client1 client0 :No such nick/channel
:client2 NICK client0
:AfterLife 433 client0  :Nickname is already in use
:client1!user1@10.14.3.10 PRIVMSG client0 :if you see this, it means that the nick was changed successfully
